lib/cli/src/commands/run/wasi.rs

```diff
@@ -102,6 +103,10 @@ pub struct Wasi {
     #[clap(long = "net")]
     pub networking: bool,
 
+    /// Define a set of filters to control the network sandbox
+    #[clap(long, requires = "networking")]
```

Comment:
Does `--net-filter` implicitly also enable networking (`--net`)?
Makes sense to do so, but should be documented.

Reply:
No the `requires` attribute will make it throw an error if the user did not also pass `--net` since a ruleset without a network permission does not make sense.